kDrive/AppDelegate.swift

```diff
@@ -74,6 +74,22 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AccountManagerDelegate, U
             UIView.setAnimationsEnabled(false)
         }
 
+        if #available(iOS 13.0, *) {
+            let defaults = UserDefaults.standard
+            let theme = defaults.object(forKey: "Theme") as? String
```

Comment:
Create a new computed variable in `DriveUserDefaults+Extension.swift` and use an enum